.github/workflows/build.yml

```diff
@@ -62,14 +62,18 @@ jobs:
           registry-url: "https://npm.pkg.github.com"
           scope: ${{ inputs.registry_scope }}
 
-      - name: Install Dependencies
+      - uses: actions/setup-node@v3.4.1
+        if: steps.s3-cache.outputs.processed == 'false'
```

Comment:
The [exploration PR](https://github.com/pleo-io/product-web/pull/9390) had a `if: inputs.skip-node == 'false' && steps.s3-cache.outputs.processed == 'false'` condition here, but I couldn't see the need to skip setting up node.